### ISSUE TITLE
Paths get parsed wrong

### ISSUE THREAD

**Issue by Achder** [zuffik/cucumber-generator#32]
Hi, 
I'm on Windows 10 and I'm getting the following error:

```
$ yarn steps
yarn run v1.22.4
$ yarn cucumber-generator --template cucumber -f features --relativePathToFeatures ./ --output file
$ C:\Development\testing\cucumber_datalab\node_modules\.bin\cucumber-generator --template cucumber -f features --relativePathToFeatures ./ --output file

Found files:
:/Development/testing/cucumber_datalab/features/login.feature
events.js:292
      throw er; // Unhandled 'error' event
      ^

Error: ENOENT: no such file or directory, open 'C:\Development\testing\cucumber_datalab\features\:\Development\testing\cucumber_datalab\features\login.feature'
Emitted 'error' event on ReadStream instance at:
    at internal/fs/streams.js:162:14
    at FSReqCallback.oncomplete (fs.js:163:23) {
  errno: -4058,
  code: 'ENOENT',
  syscall: 'open',
  path: 'C:\\Development\\testing\\cucumber_datalab\\features\\:\\Development\\testing\\cucumber_datalab\\features\\login.feature'
}
```

It looks like the paths are joined incorrect.

**Comment by zuffik** [comment]
Try to explicitly define `--outputDirectory [directory]` (eg. `--outputDirectory steps`).

**Comment by Achder** [comment]
Still the same error

**Comment by zuffik** [comment]
Okay, it is functional on MacOS and I will check it on Windows this weekend.

**Comment by zuffik** [comment]
Okay sorry for the delay, it should be fixed in #33. Please try if the fix works.
